src/test/java/org/fourhats/empleartec/tenis/StringCalculatorTests.java

```diff
@@ -0,0 +1,28 @@
+package org.fourhats.empleartec.tenis;
+import org.fourhats.empleartec.StringCalculator;
+import org.junit.Assert;
+import org.junit.Test;
+public class StringCalculatorTests {
+    @Test
+    public void debeConvertirRango() {
+        chequear("1,-2", -1);
+        chequear("1,2 , 3", 6);
+        chequear("", 0);
+        chequear("1,1", 2);
+        chequear("1,1,2", 4);
+        chequear("1000,1", 1001);
+        chequear("1,2,3,4,5,6,7,8,9", 45);
+     }
+    public void chequear(String miarray, int esperado) {
+        //Given
+        StringCalculator suma = new StringCalculator();
+        //When
+        int numeroSuma = suma.toSuma(miarray);
+        //Then
+        if(esperado == 0){

```

Comment:
please remove this if from the test, and add the case where esperado == 0, what should happens on that case?